kitchen-tests/cookbooks/end_to_end/recipes/_chef-vault.rb

```diff
@@ -33,11 +33,11 @@
 
 ruby_block "load vault item" do
   block do
-    begin
-      chef_vault_item("creds", "super_secret_1")
-    rescue ChefVault::Exceptions::SecretDecryption
-      puts "Not authorized for this key!"
-    end
+
+    chef_vault_item("creds", "super_secret_1")
+  rescue ChefVault::Exceptions::SecretDecryption
+    puts "Not authorized for this key!"
+
```

Comment:
Too bad it doesn't remove the extra new lines too.

Reply:
Yeah the lines are kinda odd here.